dask_cuda/VERSION

```diff
@@ -0,0 +1 @@
+../VERSION
```

Comment:
The Github UI makes this confusing. You don't want a file containing the text `../VERSION`, this file should be a symlink to the top-level VERSION file.

Reply:
This is how GH renders symlinks unfortunately, but good to confirm that this is a symlink

Reply:
You can tell that it wasn't a symlink before based on the CI failure.

Reply:
Thanks for the clarification @vyasr & @jakirkham ! I think I fixed it this time.